power_grid_model_io_native_c/power_grid_model_io_native/CMakeLists.txt

```diff
@@ -2,8 +2,10 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
-add_library(power_grid_model_io_native INTERFACE)
+add_library(power_grid_model_io_native STATIC) #static
```

Comment:
is there a particular reason you opted for a static library instead of an interface library here?